sdk/search/Azure.Search.Documents/src/Indexes/Models/IndexingParametersConfiguration.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Indexes.Models
+{
+    public partial class IndexingParametersConfiguration
+    {
+        private const string QueryTimeoutFormat = @"hh\:mm\:ss";
+
+        [CodeGenMember("QueryTimeout")]
+        private string _queryTimeout;
+
+        /// <summary>
+        /// Increases the timeout beyond the 5-minute default for Azure SQL database data sources.
+        /// </summary>
+        public TimeSpan? QueryTimeout
```

Comment:
Did you hit https://github.com/Azure/autorest.csharp/issues/824?

Reply:
No, this worked fine. I can't quite tell from the bug, but maybe I avoided it because it was just a string before (in the format "hh:mm:ss"). I regenerated after adding `[CodeGenMember("QueryTimeout")]` to a string field and everything went fine.